Makefile

```diff
@@ -75,3 +73,17 @@ uninstall:
 	@echo $@
 	@rm -f $(addprefix $(DESTDIR)/bin/,$(notdir $(BINARY_NAME)))
 	@rm -f $(addprefix $(DESTDIR)/bin/,$(notdir $(CLI_BINARY_NAME)))
+	
+.PHONY: check \
```

Comment:
Could we put the .PHONY near the target like

```
.PHONY: uninstall
uninstall: ....
```

 It maybe contain duplicate `.PHONY`, but we know which target is based on .PHONY. How about this?

Reply:
If we put all .PHONY targets together, we can recognize what target it contains quicker and easier instead of scanning the whole Makefile

Reply:
Thanks for the contributions. I think both are OK for me. They all have benifits. While to be honest, if I have to choose one from all together or separated, I am afraid I will choose separated. 
Thanks again and feel free to tell us if you have different opinions.  @Runrioter

Reply:
Both are OK. I made it separated. Just review it, please. @allencloud